README.md

```diff
@@ -80,6 +81,17 @@ See below for how to configure the environment variables appropriately
 - Barry__ApiKey=XXXXX # See below Barry Access Token section
 - Barry__MPID=YYYYY # See below Barry MPID section
 ```
+### Energinet
+
+```yaml
+- TeslaMate__EnergyProvider=Energinet
+- Energinet__Region=YYYYY # See below a list of all Energinet regions
+- Energinet__VAT=1.25 # VAT multiplier. In this example 25%
+- Energinet__FixedPrice__TimeZone=Europe/Copenhagen # IANA (tz database) time zone code, used for below times 
+- Energinet__FixedPrices__Prices__0=00:00-17:00=0.1432 # You can have as many as these as you need
```

Comment:
would those fixed prices be added together with `TeslaMate__FeePerKilowattHour`?
Maybe it should be stated more clearly.

Reply:
Fixed price is without VAT and added to the raw energy price from the API, then the VAT multiplier is applied:
[link](https://github.com/marval/TeslaMateAgile/blob/9260d763f378cfc91fecfff8a6802ac0e1056deb/TeslaMateAgile/Services/EnerginetService.cs#L49)

FeePerKilowattHour is used in PriceHelper and should be incl. VAT: [link](https://github.com/marval/TeslaMateAgile/blob/507bf9b9452306be0c2d284b0579f0e775daaab2/TeslaMateAgile/Helpers/PriceHelper.cs#L107)